Source/CrossingDetector/CrossingDetector.h

```diff
@@ -86,21 +86,34 @@ class CrossingDetector : public GenericProcessor
 
     void setParameter(int parameterIndex, float newValue) override;
 
+    bool enable() override;
```

Comment:
I added an override of this (which is called when acquisition starts) in order to update `eventDurationSamp` and `timeoutSamp` based on the sample rate. I like this better than re-converting `eventDuration` and `timeout` to samples each time we need them, but I still don't feel like it's 100% clean.